lib/explorer/polars_backend/data_frame.ex

```diff
@@ -858,6 +858,10 @@ defmodule Explorer.PolarsBackend.DataFrame do
   # Inspect
 
   @impl true
+  def inspect(df, opts) when node(df.data.resource) != node() do
+    Explorer.Backend.DataFrame.inspect(df, "Polars", nil, opts, from_another_node: true)
```

Comment:
```suggestion
    Explorer.Backend.DataFrame.inspect(df, "Polars", "node: #{df.data.resource}", opts, elide_columns: true)
```

Reply:
This will generate this error:

```elixir
iex(test_0@myhost)1> df = Explorer.Datasets.fossil_fuels()                                                                                                   
#Explorer.DataFrame<                                                                                                                                                
  Polars[1094 x 10]                                                                                                                                                 
  year integer [2010, 2010, 2010, 2010, 2010, ...]                                                                                                                  
  country string ["AFGHANISTAN", "ALBANIA", "ALGERIA", "ANDORRA", "ANGOLA", ...]                                                                                    
  total integer [2308, 1254, 32500, 141, 7924, ...]                                                                                                                 
  solid_fuel integer [627, 117, 332, 0, 0, ...]                                                                                                                     
  liquid_fuel integer [1601, 953, 12381, 141, 3649, ...]                                                                                                            
  gas_fuel integer [74, 7, 14565, 0, 374, ...]                                                                                                                      
  cement integer [5, 177, 2598, 0, 204, ...]                                                                                                                        
  gas_flaring integer [0, 0, 2623, 0, 3697, ...]
  per_capita f64 [0.08, 0.43, 0.9, 1.68, 0.37, ...]
  bunker_fuels integer [9, 7, 663, 0, 321, ...]
>

iex(test_0@myhost)2> "node: #{df.data.resource}"
** (Protocol.UndefinedError) protocol String.Chars not implemented for #Reference<0.641693014.1003356202.157361> of type Reference. This protocol is implemented for the following type(s): Atom, BitString, Complex, Date, DateTime, Explorer.Duration, Float, Hex.Solver.Assignment, Hex.Solver.Constraints.Empty, Hex.Solver.Constraints.Range, Hex.Solver.Constraints.Union, Hex.Solver.Incompatibility, Hex.Solver.PackageRange, Hex.Solver.Term, Integer, List, NaiveDateTime, Time, URI, Version, Version.Requirement
    (elixir 1.14.0) lib/string/chars.ex:3: String.Chars.impl_for!/1
    (elixir 1.14.0) lib/string/chars.ex:22: String.Chars.to_string/1
    iex:2: (file)
```

Reply:
Sorry, we need to inspect the node, but the overall idea should be solid. :D

Reply:
https://github.com/elixir-explorer/explorer/pull/784/commits/65116851728f66b5a2e3a734e958eda66e4e274a